<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DnD 5E LaTeX Monster Creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="twocolumn">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>567</width>
           <height>507</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="inputs">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLineEdit" name="monsterName">
              <property name="toolTip">
               <string>Monster Name</string>
              </property>
              <property name="maxLength">
               <number>64</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>Monster Foo</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="monsterType">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="toolTip">
               <string>Monster Type And Alignment</string>
              </property>
              <property name="maxLength">
               <number>64</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>Medium aberration (metasyntatic variable), neutral evil</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="basics" columnstretch="5,0,0">
              <item row="1" column="2">
               <widget class="QLineEdit" name="monsterSpeed">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>30 ft., fly 30 ft.</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_sp">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>Speed</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="monsterAC">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>15 (Leather Armor, Shield)</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_hp">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>Hit Points</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="monsterHP">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>3d8 + 3</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_ac">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>Armor Class</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="stats">
              <item row="0" column="3">
               <widget class="QLabel" name="label_int">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>INT</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="monsterSTR">
                <property name="toolTip">
                 <string>Strength</string>
                </property>
                <property name="maxLength">
                 <number>5</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>12</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="monsterCON">
                <property name="toolTip">
                 <string>Constitution</string>
                </property>
                <property name="maxLength">
                 <number>5</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>13</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_str">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>STR</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLineEdit" name="monsterCHA">
                <property name="toolTip">
                 <string>Charisma</string>
                </property>
                <property name="maxLength">
                 <number>5</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>15</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_dex">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>DEX</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_wis">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>WIS</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="monsterINT">
                <property name="toolTip">
                 <string>Intelligence</string>
                </property>
                <property name="maxLength">
                 <number>5</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>10</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="monsterDEX">
                <property name="toolTip">
                 <string>Dexterity</string>
                </property>
                <property name="maxLength">
                 <number>5</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>8</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLineEdit" name="monsterWIS">
                <property name="toolTip">
                 <string>Wisdom</string>
                </property>
                <property name="maxLength">
                 <number>5</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="placeholderText">
                 <string>14</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLabel" name="label_cha">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>CHA</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_con">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>18</height>
                 </size>
                </property>
                <property name="text">
                 <string>CON</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="details">
              <item row="0" column="0">
               <widget class="QLabel" name="label_savingthrows">
                <property name="text">
                 <string>Saving Throws</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="monsterSavingThrows">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>Str +0, Dex +0, Con +0, Int +0, Wis +0, Cha +0</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_skills">
                <property name="text">
                 <string>Skills</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_damagevulnerabilities">
                <property name="text">
                 <string>Damage Vulnerabilities</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_damageresistances">
                <property name="text">
                 <string>Damage Resistances</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_damageimmunities">
                <property name="text">
                 <string>Damage Immunities</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_conditionimmunities">
                <property name="text">
                 <string>Condition Immunities</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_senses">
                <property name="text">
                 <string>Senses</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_languages">
                <property name="text">
                 <string>Languages</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_challenge">
                <property name="text">
                 <string>Challenge</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="monsterSkills">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>Acrobatics +0, Animal Handling +0, Arcana +0, Athletics +0, Deception +0, History +0, Insight +0, Intimidation +0, Investigation +0, Medicine +0, Nature +0, Perception +0, Performance +0, Persuasion +0, Religion +0, Sleight of Hand +0, Stealth +0, Survival +0</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="monsterDamageVulnerabilities">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>cold</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="monsterDamageResistances">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>bludgeoning, piercing, and slashing from nonmagical attacks</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="monsterDamageImmunities">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>poison</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="monsterConditionImmunities">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>poisoned</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="monsterSenses">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>darkvision 60 ft., passive Perception 10</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLineEdit" name="monsterLanguages">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>Common, Goblin, Undercommon</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLineEdit" name="monsterChallenge">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="placeholderText">
                 <string>1</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="latexOut">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>latexOut</tabstop>
  <tabstop>monsterName</tabstop>
  <tabstop>monsterType</tabstop>
  <tabstop>monsterAC</tabstop>
  <tabstop>monsterHP</tabstop>
  <tabstop>monsterSpeed</tabstop>
  <tabstop>monsterSTR</tabstop>
  <tabstop>monsterDEX</tabstop>
  <tabstop>monsterCON</tabstop>
  <tabstop>monsterINT</tabstop>
  <tabstop>monsterWIS</tabstop>
  <tabstop>monsterCHA</tabstop>
  <tabstop>monsterSavingThrows</tabstop>
  <tabstop>monsterSkills</tabstop>
  <tabstop>monsterDamageVulnerabilities</tabstop>
  <tabstop>monsterDamageResistances</tabstop>
  <tabstop>monsterDamageImmunities</tabstop>
  <tabstop>monsterConditionImmunities</tabstop>
  <tabstop>monsterSenses</tabstop>
  <tabstop>monsterLanguages</tabstop>
  <tabstop>monsterChallenge</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
